### PR TITLE
add support for stack policies (create and override)

### PR DESCRIPTION
**Pull request by qwcode**
@kmanning @jantman

### CODE REVIEW

**Comment by jantman** [comment]
LGTM. :shipit: I'd like someone more familiar with using cf_deployer to check this over too, though, as I've never used it for anything.

**Comment by qwcode** [comment]
the travis failure looks like travis itself or a ruby gem thing, not the changee.   I don't see the little icon to rebuild.  I thought travis had that?

File: DETAILS.md [hunk 1]
```diff
@@ -82,7 +82,15 @@ Used by the gem for blue/green deployments and naming conventions
     * keep-previous-stack (True/False: for Cname-Swap and Auto Scaling Group Swap, previous stack will be kept after new stack is created by default. Set it to false to delete the previous stack)
     * raise-error-for-unused-inputs (True/False: it is false by default. If it is set to true, errors will be thrown if there are any inputs which are not used as parameters of CloudFormation json templates. If it is set to false or the setting does not exist, warnings will be printed in the console if there are un-used inputs.)
     * auto-scaling-group-name-output
-  * **For Components Using the Cname-Swap Deployment Strategy**
+    * create-stack-policy-filename: the name of the json file (w/o the json
```
Comment (qwcode) [inline DETAILS.md]: thinking these setting shouldn't have "filename" in the name.  to be consistent with how cf_deploy work, the file'ness and location of these should be inferred relative to the config dir.   change coming up shortly to remove "-filename" from the setting names...

**Comment by kmanning** [comment]
Looks good

**Pull request merged**